CONTRIBUTING.md

```diff
@@ -0,0 +1,102 @@
+make sure that:
+- If you are adding new functionality, create a unit test for it or update the existing tests when applicable.
+- All of your commits are atomic (one feature per commit).
+- Write a clear log message for the commit explaining the changes included in it.
+  ```bash
+  git commit -m "Summary description of the changes"
+  > Detailed list if many
+  > Detailed paragraph if the change is big and needs further explanation"
+  ```
+- Run all the tests locally in your IDE or via `gradle test`.
+- Check the Wercker CLI pipeline succeeded with your last changes.
+- You have merged the last version of `master` into your branch.
+
+Now you are ready to open a PR by following this considerations:
+- Include a description of your changes and approach taken, if many create a list of changes.
+- Reference the issue number you are addressing.
+- Keep it brief and clear. We won't complain about short PRs. Promise! ;)
+- Tag a project maintainer to inform about it. The project maintainers are: `*@lorenaqendro*`, `*@arya-hemanshu*`, `*@thanosbnt*`, `*@sassalley*`. We will assign the issue for the appropriate reviewer.
+
+## Unit tesing
+We would like to make sure that the Digital Connector works properly and if any bugs are introduced figure it out as 
+soon as possible. We know that a full test coverage is not realistic, but we'd like to cover as much as we can. 
+Therefore we will accept only PRs that include unit tests for new features or alterations of existing ones when needed.
+A reminder of this is included in the checklist of the PR template.
+  
+We use the [JUnit](http://junit.org/junit5/) framework. If you are new to it, you can find a usage guide 
+[here](http://www.vogella.com/tutorials/JUnit/article.html) as well as in action in the project test directory 
+[src/test/java](https://github.com/FutureCitiesCatapult/TomboloDigitalConnector/tree/master/src/test).
+
+## Testing
+Wercker CLI is configured to run tests on GitHub, so when you open a PR the test suite will run automatically as a build
+ which you can view in the merge box of the pull request. You can see the test output by clicking the `Details` link 
+ next to the build.
+
+To run the test suite locally, use `gradle test` from the project root. If you use IntelliJ JUnit test runner, find the 
+configuration [here](README.md#run-tests).
+
+## Conventions
+We don't like to impose strict code conventions, but we must think about the people that will read and eventually 
+change it in the future. Try to follow as much as possible the 
+ [Java code conventions](http://www.oracle.com/technetwork/java/codeconvtoc-136057.html), you can't go wrong with 
+ them. As a general rule read the suggested conventions, follow what's already embedded in the code and most 
+ importantly, use common sense.  The following guidelines suggest some minimal effort to write a clear and readable 
+ code.  
+
+- Code
+  - Indent with 4 spaces, not tabs.
```

Comment:
I think as long as tabs are 4 spaces, it doesn't really matter what they use, as they can set number of spaces of a tab press